.vscode/settings.json

```diff
@@ -2,5 +2,8 @@
   "files.associations": {
     "*.overlay": "dts",
     "*.keymap": "dts"
+  },
+  "editor.codeActionsOnSave": {
```

Comment:
I'm a bit on the fence about turning this on for everyone using VSCode... But this would likely help us avoid issues in PRs from folks.

Reply:
Happy to remove/change it.  But yes, I turned it on for that reason.

Note, I turned it on for *everything*, whereas it can be more gradual.  Perhaps just `eslint` or `prettier` etc.

I should probably have it as a separate commit.

Reply:
From reading around, I'm also getting the impression it could introduce unexpected corrections in its current form.

Reply:
I've removed that setting for now.  It can go in workspace settings until we've experimented and found the optimum approach.